oaknational/Oak-Web-Application
# Fix/expired lessons page

## Description

for expired lessons, navigating to the expired lesson will not show lesson data, but will have a message saying lesson is expired.

## Issue(s)

Fixes #1352 

## How to test

1. Go to http://localhost:3000/beta/teachers/key-stages/ks3/subjects/english/units/the-sonnet-ballad-gwendolyn-brooks-3e19/lessons/the-sonnet-ballad-lesson-1-of-5-68w3gt
3. You should see that the expired lesson is no longer available to view.

## Screenshots

How it used to look (delete if n/a):
<img width="1411" alt="Screenshot 2023-03-27 at 16 01 05" src="https://user-images.githubusercontent.com/48293828/227980141-ab980e8f-cd81-46ba-8a35-f06d57df9e14.png">


How it should now look:
<img width="1124" alt="Screenshot 2023-03-27 at 16 00 41" src="https://user-images.githubusercontent.com/48293828/227980019-d80b3798-d9e0-47ac-97b2-0419ebd3bea6.png">


## Checklist
